src/components/EditContactInformation/EditContactInformation.jsx

```diff
@@ -0,0 +1,181 @@
+// type,
+//       name,
+//       street,
+//       zipCode,
+//       state,
+//       qbVendorName,
+//       qbCityStateZip,
+//       primaryPhone,
+//       backupPhone,
+//       primaryEmail,
+//       comments,
+//       faxPhone,
+//       contactName,
+//       website,
+//       businessHours,
+//       findOut,
+//       onboardingStatus,
+//       joinDate,
+//       inputTypeStatus,
+//       vendorType,
+//       status,
+//       petsOfTheHomelessDiscount,
+//       updatedBy,
+//       updatedDateTime,
+//       syncToQb,
+//       veterinary,
+//       resource,
+//       food,
+//       donation,
+//       familyShelter,
+//       wellness,
+//       spayNeuter,
+//       financial,
+//       reHome,
+//       erBoarding,
+//       senior,
+//       cancer,
+//       dog,
+//       cat,
+//       fphPhone,
+//       contactPhone,
+//       webNotes,
+//       internalNotes,
+//       published,
+//       shelter,
+//       domesticViolence,
+//       webDateInit,
+//       entQb,
+//       serviceRequest,
+//       inactive,
+//       finalCheck,
+//       createdBy,
+//       createdDate,
+//       city,
+
+import { useBackend } from '../../contexts/BackendContext';
+import { useState, useEffect } from 'react';
+import { Flex, Text, Textarea, Stack, Box, Spacer, Button, Input, InputGroup, InputLeftElement, Icon } from '@chakra-ui/react';
+import { PhoneIcon } from '@chakra-ui/icons';
+
+
+const EditContactInformation = () => { // how will id be passed in?
+    const { backend } = useBackend();
+    const [name, setName] = useState();
+    const [phoneNumber, setPhoneNumber] = useState();
+    const [email, setEmail] = useState();
+    const [website, setWebsite] = useState();
+    const [address, setAddress] = useState();
+    const [businessName, setBusinessName] = useState();
+    const [formData, setFormData] = useState(null);
+
+    const updateForm = () => {
+      setFormData({
+        id: 0, // hard-code right now
+        name: businessName,
+        primary_phone: phoneNumber,
+        primary_email: email,
+        website: website,
+        street: address,
+        contact_name: name,
+      });
+    };
+
+    const getBusinessInfo = async () => {
+        const result = await backend.get('/business/0');
+        setName(result.data[0].contact_name);
+        setPhoneNumber(result.data[0].primary_phone);
+        setEmail(result.data[0].primary_email);
+        setWebsite(result.data[0].website);
+        setAddress(result.data[0].street);
+        setBusinessName(result.data[0].name);
+    };
+
+
+
+    useEffect(() => {
+      const saveBusinessInfo = async () => {
+        try {
+            if (formData) {
+              const result = await backend.put('/business/0', formData );
+              console.log(result);
+            }
+        } catch (err) {
+            console.log(err);
+        }
+      };
+      saveBusinessInfo();
+    }, [formData]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'backend'. Either include it or remove the dependency array.